messaging/src/main/java/org/axonframework/eventhandling/TrackingEventProcessor.java

```diff
@@ -175,6 +177,11 @@ protected TrackingEventProcessor(Builder builder) {
             }
             return interceptorChain.proceed();
         });
+
+        registerHandlerInterceptor((unitOfWork, interceptorChain) -> spanFactory
```

Comment:
Curious: why haven't you done this for the `SubscribingEventProcessor`?

Reply:
Because it's directly done in the publish span. Every action taken there is thus already part of the trace automatically, and the event name is already in the span as well

Reply:
Resolving this comment as we've discussed the point you've shared above and came to that conclusion.